baseframe/static/js/baseframe-material.js

```diff
@@ -370,6 +353,12 @@ window.Baseframe.MapMarker.prototype.getDefaultLocation = function() {
   return {latitude: latitude, longitude: longitude};
 };
 
+window.Baseframe.MapMarker.prototype.activate = function(){
+  var self = this;
```

Comment:
`self` is not being used in this function. What's it for?

Reply:
Removed